server/routes/shared/roshPanelView.ts

```diff
@@ -17,26 +17,40 @@ export default class RoshPanelView {
         return [
           { text: utils.convertToProperCase(row.riskTo) },
           {
-            text: tagMacro({
-              text: row.riskScore.replace('_', ' '),
-              classes: this.tagClassForRiskScore(row.riskScore),
-            }),
+            html: `<span class="rosh-analysis-table__risk-score ${this.classForRiskScore(
+              row.riskScore
+            )}">${this.stringForRiskScore(row.riskScore)}</span>`,
           },
         ]
       }),
     }
   }
 
-  private tagClassForRiskScore(riskScore: string): string {
+  private classForRiskScore(riskScore: string): string {
     switch (riskScore) {
       case 'LOW':
-        return 'govuk-tag--green'
+        return 'rosh-analysis-table__risk-score__low'
       case 'MEDIUM':
-        return 'govuk-tag--blue'
+        return 'rosh-analysis-table__risk-score__medium'
       case 'HIGH':
-        return 'govuk-tag--red'
+        return 'rosh-analysis-table__risk-score__high'
       case 'VERY_HIGH':
-        return 'govuk-tag--purple'
+        return 'rosh-analysis-table__risk-score__very-high'
+      default:
+        return ''
+    }
+  }
+
+  private stringForRiskScore(riskScore: string): string {
```

Comment:
nit: it worries me we may end up not displaying any score at all if say, suddently the 'VERY_HIGH' score was returned  as 'VERY HIGH' (i know, not super realistic).. wonder if we can use some rules to convert the input string instead of this switch. e.g. replace '_' with space, and then 'proper case' the result